internal/pkg/cli/app_init.go

```diff
@@ -209,30 +209,30 @@ func (opts *InitAppOpts) listDockerfiles() ([]string, error) {
 	if err != nil {
 		return nil, fmt.Errorf("read directory: %w", err)
 	}
-	var dockerfiles []string
+	var directories []string
 
 	for _, wdFile := range wdFiles {
 		// Add Dockerfiles in current directory, otherwise continue.
 		if !wdFile.IsDir() {
 			if wdFile.Name() == "Dockerfile" {
-				dockerfiles = append(dockerfiles, filepath.Join(".", wdFile.Name()))
+				directories = append(directories, ".")
 			}
 			continue
 		}
 
 		// Add Dockerfiles in one sub-directory below.
-		subFiles, err := afero.ReadDir(opts.fs, filepath.Join(".", wdFile.Name()))
+		subFiles, err := afero.ReadDir(opts.fs, wdFile.Name())
 		if err != nil {
 			return nil, fmt.Errorf("read directory: %w", err)
 		}
 		for _, f := range subFiles {
 			if f.Name() == "Dockerfile" {
-				dockerfiles = append(dockerfiles, filepath.Join(".", wdFile.Name(), "Dockerfile"))
+				directories = append(directories, "./"+wdFile.Name())
 			}
 		}
 	}
-	sort.Strings(dockerfiles)
-	return dockerfiles, nil
+	sort.Strings(directories)
+	return directories, nil
```

Comment:
Can we update the method's comment to reflect that it now returns directories instead?

Why are we making this change?  
Is it possible to update `AppInit` to add the "/Dockerfile" at the end of the options? to keep the current UX

Reply:
We're making the change to support `docker build` command.

`docker build` expects a directory to build an image. For example `docker build .` will build the `Dockerfile` in the current directory.

`docker build ./frontend` will build the `Dockerfile` in the frontend folder.

`docker build Dockerfile` or `docker build frontend/Dockerfile` is invalid.

Reply:
#275 to follow up on user experience when asked to select from a list of found Dockerfiles.